application.properties fails to load on start up

When opening a project (ex: config-quickstart), there is the following request failure after opening the application.properties file.

```
WARNING: Error while getting MicroProfileProjectInfo (classpath) for '.../application.properties'
java.util.concurrent.CompletionException: org.eclipse.lsp4j.jsonrpc.ResponseErrorException: Request microprofile/projectInfo failed with message: class java.lang.NoSuchMethodError cannot be cast to class java.lang.Exception (java.lang.NoSuchMethodError and java.lang.Exception are in module java.base of loader 'bootstrap')
```

I see it as well on latest versions of the language server :

```
[Trace - 12:32:21 p.m.] Sending response 'microprofile/projectInfo - (10)'. Processing request took 543ms
Jul. 04, 2022 12:32:21 P.M. org.eclipse.lsp4mp.ls.properties.MicroProfileProjectInfoCache lambda$getProjectInfo$0
WARNING: Error while getting MicroProfileProjectInfo (classpath) for 'file:///home/rgrunber/sample-projects/quarkus-quickstarts/config-quickstart/src/main/resources/application.properties'
java.util.concurrent.CompletionException: org.eclipse.lsp4j.jsonrpc.ResponseErrorException: Request microprofile/projectInfo failed with message: class java.lang.NoSuchMethodError cannot be cast to class java.lang.Exception (java.lang.NoSuchMethodError and java.lang.Exception are in module java.base of loader 'bootstrap')
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(Unknown Source)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleResponse(RemoteEndpoint.java:209)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:193)
	at org.eclipse.lsp4mp.ls.commons.ParentProcessWatcher.lambda$apply$0(ParentProcessWatcher.java:148)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.eclipse.lsp4j.jsonrpc.ResponseErrorException: Request microprofile/projectInfo failed with message: class java.lang.NoSuchMethodError cannot be cast to class java.lang.Exception (java.lang.NoSuchMethodError and java.lang.Exception are in module java.base of loader 'bootstrap')
	... 11 more
```

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -1,7 +1,7 @@
 pipeline {
   agent any
   tools {
-    jdk 'adoptopenjdk-hotspot-jdk11-latest'
+    jdk 'temurin-jdk17-latest'
   }
   environment {
     MAVEN_HOME = "$WORKSPACE/.m2/"

--- a/Release.Jenkinsfile
+++ b/Release.Jenkinsfile
@@ -1,7 +1,7 @@
 pipeline {
   agent any
   tools {
-    jdk 'adoptopenjdk-hotspot-jdk11-latest'
+    jdk 'temurin-jdk17-latest'
   }
   environment {
     MAVEN_HOME = "$WORKSPACE/.m2/"

--- a/microprofile.jdt/org.eclipse.lsp4mp.jdt.core/META-INF/MANIFEST.MF
+++ b/microprofile.jdt/org.eclipse.lsp4mp.jdt.core/META-INF/MANIFEST.MF
@@ -20,7 +20,7 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.ltk.core.refactoring,
  org.eclipse.text,
  org.apache.commons.lang3
-Bundle-RequiredExecutionEnvironment: JavaSE-11
+Bundle-RequiredExecutionEnvironment: JavaSE-17
 Bundle-ActivationPolicy: lazy
 Export-Package: io.quarkus.runtime.util,
  org.eclipse.lsp4mp.commons,

--- a/microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/core/utils/DependencyUtil.java
+++ b/microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/core/utils/DependencyUtil.java
@@ -62,7 +62,7 @@ public class DependencyUtil {
 		// Download pom
 		File pomFile = getLocalArtifactFile(key, POM_FILE_TYPE);
 		if (pomFile == null) {
-			Artifact pom = MavenPlugin.getMaven().resolve(key.getGroupId(), key.getArtifactId(), key.getVersion(),
+			Artifact pom = MavenPlugin.getMaven().resolve(key.groupId(), key.artifactId(), key.version(),
 					POM_FILE_TYPE, null, null, monitor);
 			if (pom == null) {
 				throw new FileNotFoundException("Unable to find pom file for " + key);// Should it be just a warning?
@@ -71,8 +71,8 @@ public class DependencyUtil {
 		// Download jar
 		File archive = getLocalArtifactFile(key, JAR_FILE_TYPE);
 		if (archive == null) {
-			Artifact artifact = MavenPlugin.getMaven().resolve(key.getGroupId(), key.getArtifactId(), key.getVersion(),
-					JAR_FILE_TYPE, key.getClassifier(), null, monitor);
+			Artifact artifact = MavenPlugin.getMaven().resolve(key.groupId(), key.artifactId(), key.version(),
+					JAR_FILE_TYPE, key.classifier(), null, monitor);
 			if (artifact == null) {
 				throw new FileNotFoundException("Unable to find " + key);
 			}
@@ -88,8 +88,8 @@ public class DependencyUtil {
 		IMaven maven = MavenPlugin.getMaven();
 		try {
 			ArtifactRepository localRepository = maven.getLocalRepository();
-			String relPath = maven.getArtifactPath(localRepository, a.getGroupId(), a.getArtifactId(), a.getVersion(),
-					type, a.getClassifier());
+			String relPath = maven.getArtifactPath(localRepository, a.groupId(), a.artifactId(), a.version(),
+					type, a.classifier());
 			File file = new File(localRepository.getBasedir(), relPath).getCanonicalFile();
 			if (file.canRead() && file.isFile()) {
 				return file;

--- a/microprofile.jdt/org.eclipse.lsp4mp.jdt.test/META-INF/MANIFEST.MF
+++ b/microprofile.jdt/org.eclipse.lsp4mp.jdt.test/META-INF/MANIFEST.MF
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: quarkus.jdt.ls Test Plugin
 Bundle-SymbolicName: org.eclipse.lsp4mp.jdt.test;singleton:=true
 Bundle-Version: 0.5.0.qualifier
-Bundle-RequiredExecutionEnvironment: JavaSE-11
+Bundle-RequiredExecutionEnvironment: JavaSE-17
 Require-Bundle: org.junit,
  org.apache.commons.io,
  org.apache.commons.lang3,

--- a/microprofile.jdt/pom.xml
+++ b/microprofile.jdt/pom.xml
@@ -14,7 +14,7 @@
     <tycho.extras.version>${tycho.version}</tycho.extras.version>
     <tycho.scmUrl>scm:git:https://github.com/eclipse/lsp4mp</tycho.scmUrl>
     <tycho.generateSourceReferences>true</tycho.generateSourceReferences>
-    <jdt.ls.version>1.7.0.20211216164144</jdt.ls.version>
+    <jdt.ls.version>1.13.0.20220630173130</jdt.ls.version>
     <tycho.test.platformArgs />
     <tycho.test.jvmArgs>-Xmx512m ${tycho.test.platformArgs}</tycho.test.jvmArgs>
 
@@ -36,7 +36,7 @@
     <repository>
       <id>jdt.ls.p2</id>
       <layout>p2</layout>
-      <url>https://download.eclipse.org/jdtls/milestones/1.12.0/repository/</url>
+      <url>https://download.eclipse.org/jdtls/milestones/1.13.0/repository/</url>
     </repository>
     <repository>
       <id>jdt.ls.maven</id>

